scripts/recipes/librispeech/steps/data/reverberate_data_dir.py

```diff
@@ -813,7 +813,7 @@ def parse_rir_list(rir_set_para_array, smoothing_weight, sampling_rate=None):
     return rir_list
 
 
```

Comment:
Looks like autoblack ran on the scripts/ directory for some reason... Not really sure why it wouldn't already have made these changes though.